package.json

```diff
@@ -85,6 +85,7 @@
     "morgan": "1.2.0",
     "ms": "0.7.1",
     "node-sass": "3.7.0",
+		"notifications-panel": "git+https://github.com/Automattic/notifications-panel.git",
```

Comment:
This can be shortened to:
```
"notifications-panel": "Automattic/notifications-panel#mycommitsha"
```
https://docs.npmjs.com/files/package.json#github-urls

Once this gets closer I'd recommend pointing at a specific tag or branch.